requirements.txt

```diff
@@ -43,6 +43,7 @@ slimit
 sqlparse
 pytas
 bibtexparser
+mozilla-django-oidc
```

Comment:
We might need an upper-constraint on this, given that we only support Django 1.11 at the moment. Is there a version of this module that drops support for Django 1.11/Python 2?

Reply:
> The changes look pretty simple to me. How can we differentiate between users who logged in via OpenID versus the ones who were pre-existing / haven't logged in with that method?
> 
> What about the changes for things like, is PI status approved, who approved it and when? Is that going to be in another PR?
> 
> What do you think about renaming `is_pi_eligible` to `is_pi`? If the role is on the user in Keycloak, that will meant they have been granted the status.

I'm still working on differentiating between the two types of login methods. I was planning on making that and the changes for PI approved status part of another PR. If the role in Keycloak indicates pi status has been granted, I think it makes sense to rename to `is_pi`

Reply:
Adding a constraint to less than the next dot release sounds like a sane plan!